spec/models/service_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+require 'securerandom'
+
+FactoryBot.define do
+  factory :service do
+    entity_id { SecureRandom.alphanumeric }
```

Comment:
entity_id will be an URI and we might want to enforce/validate it later, so might be better using a hardcoded test value instead of `SecureRandom`